Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -1096,6 +1096,23 @@ private static void RunInitializeInPlayer()
 #if UNITY_EDITOR
         private static InputSystemObject s_SystemObject;
 
+        private static void CheckCurrentPlayerSettingForInputSystem()
```

Comment:
Method names like this are bad as it doesn't really tell what the method does. Check how? And do what with the check? (given there is no return value to indicate the result of the check)

Reply:
Also recommend moving the method into `InputSystemObject`.